eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-devops.yml

```diff
@@ -0,0 +1,99 @@
+resources:
```

Comment:
Why are we putting this under the jobs templates? This looks like an entire pipeline correct?

Reply:
If the top level work declaration is jobs and not stages, I think it still belongs in jobs, unless it should go in `eng/pipelines` instead of `templates`.